ProcessLib/ThermoMechanicalPhaseField/Tests.cmake

```diff
@@ -0,0 +1,27 @@
+AddTest(
```

Comment:
In the PR description you mention four tests, but there are only two ctests: 3D_non-isothermal and 3D_beam_LARGE. Something is missing....

Reply:
other two tests including the IGLU TES and thermal shock slab are added in ctests.